app/controllers/idv/document_capture_controller.rb

```diff
@@ -50,6 +50,7 @@ def extra_view_variables
       }.merge(
         acuant_sdk_upgrade_a_b_testing_variables,
         phone_question_ab_test_analytics_bucket,
+        phone_with_camera: idv_session.phone_with_camera,
```

Comment:
for readability .. insert on line 50 instead?

{
        document_capture_session_uuid: document_capture_session_uuid,
        flow_path: 'standard',
        sp_name: decorated_sp_session.sp_name,
        failure_to_proof_url: return_to_sp_failure_to_proof_url(step: 'document_capture'),
        phone_with_camera: idv_session.phone_with_camera,
      }.merge(
        acuant_sdk_upgrade_a_b_testing_variables,
        phone_question_ab_test_analytics_bucket,
     )

Reply:
[Done](https://github.com/18F/identity-idp/pull/9481/commits/7523b829b1c134dfd08de3c5b7dcc8dd72d51694)